server/config/client.go

```diff
@@ -55,7 +55,8 @@ func GenerateClientConfig(c *model.Config, telemetryID string, license *model.Li
 	// This setting is only temporary, so keep using the old setting name for the mobile and web apps
 	props["ExperimentalEnablePostMetadata"] = "true"
 
-	props["EnableAppBar"] = strconv.FormatBool(*c.ExperimentalSettings.EnableAppBar)
+	props["EnableAppBar"] = strconv.FormatBool(!*c.ExperimentalSettings.DisableAppBar)
+	props["DisableAppBar"] = strconv.FormatBool(*c.ExperimentalSettings.DisableAppBar)
```

Comment:
Keeping the old setting in the returned client config. To my knowledge, this isn't used by anything outside of the webapp, so this may not be necessary. Please see this discussion for more info: https://github.com/mattermost/mattermost-server/pull/21715/files#r1030916618

Reply:
if we keep the EnableAppBar prop, it could be a bit surprising after some months. 

Could you please add a clarifying comment?

Reply:
@trilopin The way the feature is set up, and the backwards compatible nature of the migration from the channel header to the Apps Bar, plugins do need to and should not be referencing this variable. This setting started out as a feature flag, but was moved to a config setting so customers could turn it on for Calls.

I'm thinking we should just get rid of it. We don't have any code referencing this once this PR is merged. I doubt any 3rd party code is referencing this variable, but it's possible.